emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
+             "end_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of ending point
+             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
```

Comment:
I am not sure what you mean by this. Do you mean that are no place objects? There definitely are, that's what the Place wrapper class is about.